contracts/SpokePool.sol

```diff
@@ -225,16 +225,16 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
         });
         bytes32 relayHash = _getRelayHash(relayData);
 
-        // Check that the caller is filling a non zero amount of the relay and the relay has not already been completely
-        // filled. Note that the `relays` mapping will point to the amount filled so far for a particular `relayHash`,
-        // so this will start at 0 and increment with each fill.
-        require(maxTokensToSend > 0 && relayFills[relayHash] < totalRelayAmount, "Cannot send 0, or relay filled");
+        // Check that the relay has not already been completely filled. Note that the `relays` mapping will point to
+        // the amount filled so far for a particular `relayHash`, so this will start at 0 and increment with each fill.
+        require(relayFills[relayHash] < totalRelayAmount, "relay filled");
 
         // Stores the equivalent amount to be sent by the relayer before fees have been taken out.
-        uint256 fillAmountPreFees = _computeAmountPreFees(maxTokensToSend, (realizedLpFeePct + relayerFeePct));
+        uint256 fillAmountPreFees = 0;
 
         // Adding brackets "stack too deep" solidity error.
-        {
+        if (maxTokensToSend > 0) {
```

Comment:
Can you think of anywhere that we use the filledAmount stored in the contract as a proxy for the relay being "initialized"? If so, this relay would still be considered uninitialized. I can't think of anywhere this would cause problems, just wanted to ask in case you could.

Reply:
We don't, there isn't really a good way of checking whether a relay was initialized at the moment. For that we'd need to expand the data associated with each relay hash